dlhub_sdk/tests/test_dlhub_client.py

```diff
@@ -52,18 +52,19 @@ def test_get_servables(dl):
 
 
 def test_run(dl):
-    user = "zhuozhao_uchicago"
-    name = "noop"
-    data = {"data": ["V", "Co", "Zr"]}
+    user = "aristana_uchicago"
+    name = "noop_v10"
+    data = True  # accepts anything as input, but listed as Boolean in DLHub
 
     # Test a synchronous request
     res = dl.run("{}/{}".format(user, name), data, timeout=60)
-    assert res == 'Hello'
+    # res[0] contains model results, res[1] contains event data JSON
+    assert res[0] == 'Hello world!'
 
     # Test an asynchronous request
     task_id = dl.run("{}/{}".format(user, name), data, asynchronous=True)
     assert isinstance(task_id, DLHubFuture)
-    assert task_id.result(timeout=60) == 'Hello'
+    assert task_id.result(timeout=60)[0] == 'Hello world!'
```

Comment:
I think the original assertion was the correct behavior. `.run` should only return the result data and not the event JSON when you set `debug=False` (the default for `run`)

Reply:
Perhaps we should add a test where we turn debug=True to see what the difference in behavior is.